checker/src/org/checkerframework/checker/index/samelen/SameLenAnnotatedTypeFactory.java

```diff
@@ -73,6 +77,52 @@ public QualifierHierarchy createQualifierHierarchy(MultiGraphFactory factory) {
         return new SameLenQualifierHierarchy(factory);
     }
 
+    @Override
+    public AnnotatedTypeMirror getAnnotatedTypeLhs(Tree tree) {
+        AnnotatedTypeMirror atm = super.getAnnotatedTypeLhs(tree);
+        if (tree.getKind() == Tree.Kind.VARIABLE) {
+            String varName;
```

Comment:
If I did, I wouldn't be able to call it from here, because I don't have a `FieldAccessNode`, which is why I didn't write this in `FlowExpressions` in the first place.